### DISCUSSION TITLE
Polylith CI/CD - Git Flow

### DISCUSSION THREAD

**Discussion by ljubisa-dobric** [DavidVujic/python-polylith#283] [Q&A]
Hello community, I have a question if someone has experiance with polylith and deplyoing multiple projects using Git-Flow branching strategy?  Also what was some drawbacks or issues that you needed to solve would be also helpfull :)

**Comment by hstravis** [comment]
@ljubisa-dobric My experience might be relevant. We use a single `main` branch and feature branches and use tags for releases. So not exactly git flow. But we use polylith to maintain a number of different applications in a single mono repo. Polylith works great out of the box for the python and docker side of things. But it doesn't address configuration management, or deploying to Kubernetes, or test sharding - they're out of scope for polylith. So we wrote number of bash scripts for that. I could go on at length about what we did but I don't know if it is relevant. Do you have specific questions? Could you describe your project(s)?
